src/search/move_picker.h

```diff
@@ -18,17 +18,21 @@ enum KillerMoveType {
     Second,
 };
 
+enum QuietMoveOffsets {
+    KillerMoveFirst = 30003,
+    KillerMoveSecond = 30002,
+    CounterMove = 30001,
```

Comment:
what are these values based on? Maybe scale them excessively so we guarantee that we don't overflow history scores :)

Reply:
I used large values that were far away from history move scores, to force ordering.

Do you have any suggestions for alternative values? I'm not sure how we'd have an overflow here.

Reply:
Changed, but the now-removed debug statement already confirmed that history scores were always lower than the offsets.

Also started a new test just to ensure nothing's broken.

Reply:
> Changed, but the now-removed debug statement already confirmed that history scores were always lower than the offsets.

*Always* is a big word. Maybe during bench, but that's also not a deep search. 
The history scores will keep increasing during a search, so it's better to be safe than sorry :)